.gitignore

```diff
@@ -1,4 +1,29 @@
-.idea
+# IDEA/Android Studio project
+*.iml
+*.ipr
+*.iws
+.idea/
+
+# Visual Studio project
```

Comment:
uber nit: Visual Studio Code. 'Visual Studio' is actually just a Xamarin IDE

Reply:
Yup. 'Visual Studio Code' would we better. Visual Studio is an entirely different product.